app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
according to the [PEP 8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) it is better to use Exception for custom Exceptions